project-base/storefront/components/Layout/Header/MenuIconic/SalesRepresentative.tsx

```diff
@@ -21,37 +21,44 @@ export const SalesRepresentative: FC = () => {
     }
 
     return (
-        <div className="flex items-start gap-4 w-full pt-4">
-            {salesRepresentative.image && (
-                <Image
-                    alt={t('Need advice?')}
-                    className="w-12 h-12 rounded-full object-cover"
-                    height={100}
-                    src={salesRepresentative.image.url}
-                    width={100}
-                />
-            )}
-            <div className="font-medium dark:text-white w-full">
-                {fullName && (
-                    <>
-                        <p className="text-lg font-semibold font-secondary leading-7">{fullName}</p>
-                        <p className="text-xs font-semibold font-secondary text-textSubtle uppercase tracking-wider">
-                            {t('Your sales representative')}
-                        </p>
-                    </>
+        <div className="flex flex-col font-semibold">
+            <div className="flex items-start gap-4 w-full pt-4">
+                {salesRepresentative.image && (
+                    <Image
+                        alt={t('Need advice?')}
+                        className="w-12 h-12 rounded-full object-cover"
+                        height={100}
+                        src={salesRepresentative.image.url}
+                        width={100}
+                    />
                 )}
+                <div className="w-full">
+                    {fullName && (
```

Comment:
Move condition one row above to avoid an empty div when there is no `fullName`.

Reply:
Fixed